rc/windowing/detection.kak

```diff
@@ -26,16 +26,23 @@ manual control of the module loading." \
 str-list windowing_modules 'tmux' 'screen' 'kitty' 'iterm' 'wayland' 'x11'
 
 hook -group windowing global KakBegin .* %{
+    define-command missing-windowing-system -params .. -docstring \
+    'Placeholder implementation for the focus and terminal aliases' %{
+        fail Run Kakoune under a windowing system to use this command
```

Comment:
It would be more helpful to list the actual supported windowing systems, rather than making people guess whether Kakoune's definition of "windowing system" aligned with their own.

```suggestion
        fail This command only works inside a windowing system, like: %opt{windowing_modules}
```

Reply:
Good point. Done! :)